duecredit/tests/test_io.py

```diff
@@ -211,3 +212,10 @@ def test_enumeratedentries():
     assert_raises(KeyError, enumentries.fromrefnr, 666)
 
     assert_equal(entries, sorted(enumentries, key=lambda x: x[1]))
+
+def test_get_text_rendering():
+    citation_bibtex = Citation(_sample_bibtex, path='mypath')
+    citation_doi = Citation(_sample_doi, path='mypath')
+    # smoke test
+    assert_true(get_text_rendering(citation_bibtex))
```

Comment:
why just a  smoke test ?  ideally mock out format_bibtex  a nd verify that  it gets called in  both cases.  and you could even  verify that they get correct  input to them